tutormfe/plugin.py

```diff
@@ -15,34 +16,23 @@
         "CADDY_DOCKER_IMAGE": "{{ DOCKER_IMAGE_CADDY }}",
         "ACCOUNT_MFE_APP": {
             "name": "account",
-            "repository": "https://github.com/edx/frontend-app-account",
+            "repository": "https://github.com/openedx/frontend-app-account",
             "port": 1997,
-            "env": {
-                "production": {
-                    "COACHING_ENABLED": "",
-                    "ENABLE_DEMOGRAPHICS_COLLECTION": "",
-                },
-            },
         },
         "GRADEBOOK_MFE_APP": {
             "name": "gradebook",
-            "repository": "https://github.com/edx/frontend-app-gradebook",
+            "repository": "https://github.com/openedx/frontend-app-gradebook",
             "port": 1994,
         },
         "LEARNING_MFE_APP": {
             "name": "learning",
-            "repository": "https://github.com/edx/frontend-app-learning",
+            "repository": "https://github.com/openedx/frontend-app-learning",
             "port": 2000,
         },
         "PROFILE_MFE_APP": {
             "name": "profile",
-            "repository": "https://github.com/edx/frontend-app-profile",
+            "repository": "https://github.com/openedx/frontend-app-profile",
             "port": 1995,
-             "env": {
-                "production": {
-                    "ENABLE_LEARNER_RECORD_MFE": "true",
-                },
-            },
```

Comment:
Same as above, doesn't this also need to be added to LMS settings using MFE_CONFIG_OVERRIDES?

Reply:
I have talked with @arbrandes before about this, the configuration isn't necessarly, and would need to be added/setted when credential service enabled openedx/build-test-release-wg/issues/168

Reply:
Got it, thanks.